app/build.gradle

```diff
@@ -95,10 +95,13 @@ android {
         warningsAsErrors true
         abortOnError true
     }
+
+    viewBinding.enabled = true
 }
 
 androidExtensions {
     experimental = true
+    features = ["parcelize"] // unable to disable synthetics, so only allow others
```

Comment:
What are the arguments behind "disabling synthetics"?
Are there some preformance / build-time / runtime improvements, or _just_ to force the usage of ViewBinding (by disabling the usage of synthetics)?

If only the latter, I would like to discuss whether that's necessary, as this seems to me to be a pain to search for / find if one would need another feature of the androidExtensions (may also be any not yet available, upcoming feature) enabled.

One could also still use direct `findViewById`in any case.
Furthermore, should there be a _hardcut_ when upgrading an existing reaktor project to the new version, so that every view needs to be changed to not use synthetics anymore?

Reply:
> Are there some preformance / build-time / runtime improvements, or just to force the usage of ViewBinding (by disabling the usage of synthetics)?

There are no performance improvements. It's just for enforcing ViewBinding due to its advantages that we talked about in the ADM.

> If only the latter, I would like to discuss whether that's necessary, as this seems to me to be a pain to search for / find if one would need another feature of the androidExtensions (may also be any not yet available, upcoming feature) enabled.

A valid statement. Depends on what the others think. Although I actually prefer adding features instead of disabling them. Through that we have actual control over the features we want to use. As we have seen with synthetics, these features might actually cause harm if used incorrectly (thinking about `LayoutContainer`).

> Furthermore, should there be a hardcut when upgrading an existing reaktor project to the new version, so that every view needs to be changed to not use synthetics anymore?

There is a hard-cut anyway, due to the coroutines merge that happened today.

Reply:
Ok, who put the "resolve conversation" button so conveniently below the "comment-button? :sweat_smile: (or I'm just too familiar with gitlabs comment-and-resolve button :grimacing:)
Wanted to resolve this with the comment:

Sounds good to me, thanks for the clarification! (especially when we merge this somewhat soon, before more projects start using control, but not view binding :slightly_smiling_face:)